package.json

```diff
@@ -42,7 +42,7 @@
     "cy:open": "cypress open"
   },
   "dependencies": {
-    "@storyblok/richtext": "3.0.2",
+    "@storyblok/richtext": "https://pkg.pr.new/@storyblok/richtext@8a49bc6",
```

Comment:
why we need to pin this?

Reply:
This is not for review still sorry, I should add the draft state. This is only for testing purposes to be able to use pkg.pr.new to test down.